ZeeguuAPI/ZeeguuAPITests/ZeeguuAPITests.swift

```diff
@@ -0,0 +1,169 @@
+//
+//  ZeeguuAPITests.swift
+//  ZeeguuAPITests
+//
+//  Created by Jorrit Oosterhof on 28-11-15.
+//  Copyright © 2015 Jorrit Oosterhof. All rights reserved.
+//
+
+import XCTest
+@testable import ZeeguuAPI
+
+class ZeeguuAPITests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testExample() {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+		let l = NSCondition()
+		l.lock()
+		print("Testing login:")
+		ZeeguuAPI.sharedAPI().loginWithEmail("j.oosterhof.4@student.rug.nl", password: "JLq-E6q-MzL-8pp") { (success) -> Void in
+			print("success: ", success)
+			print("sessionID: ", ZeeguuAPI.sharedAPI().currentSessionID)
+			l.signal()
+		}
+		l.wait()
+		l.unlock()
+		
+		print("Testing learned language:")
+		l.lock()
+		ZeeguuAPI.sharedAPI().learnedLanguage { (langCode) -> Void in
+			print("langCode: ", langCode)
+			l.signal()
+		}
+		l.wait()
+		l.unlock()
```

Comment:
I see that the tests are working, and this is great! 
However, there are three inter-related things that would be cool to do here: 

1. remove some of the l.lock() l.unlock() which are very repetitive in the code at the moment 
2. create individual unit tests for the different endpoints, such that if there is a failure, we know which test has failed
3. use assertions instead of printing values

One hint: given that setUp() and tearDown() are executed before every test, you can declare the l in your class, lock it in setUp() and unlock it in tearDown().